react/features/speaker-stats/functions.js

```diff
@@ -191,3 +193,17 @@ export function resetHiddenStats(state: Object, stats: ?Object) {
 
     return resetStats;
 }
+
+/**
+ * Returns true if the face expressions switch should be displayed, false otherwise.
+ *
+ * @param  {Object} state - The redux state.
+ * @returns {boolean}
+ */
+export function showFaceExpressionsSwitch(state: Object) {
+    const { faceLandmarks } = state['features/base/config'];
+    const { jwt } = state['features/base/jwt'];
```

Comment:
Why the JWT check?

Reply:
To have the epxressions visible only for logged users

Reply:
I don't think we want that, or it'll be unusable for most of the users.